packages/app/src/components/DBTraceWaterfallChart.tsx

```diff
@@ -118,6 +120,14 @@ function getConfig(source: TSource, traceId: string) {
               },
             ]
           : []),
+        ...(alias.SpanAttributes
```

Comment:
I think it's preferable in general performance-wise for us to specifically pull out the values `http.url`, etc. This in theory will also help if we do things like materialized columns on those values and whatnot.